output.tf

```diff
@@ -1,4 +1,20 @@
-#output "example" {
-#  value       = var.example
-#  description = "This is just an example of an output"
-#}
+output "service_account_name" {
+  value       = local.service_account_name
+  description = "The Service Account name"
+}
+
+output "service_account_private_key" {
+  value       = length(var.service_account_private_key) > 0 ? var.service_account_private_key : module.lacework_gke_svc_account.private_key
+  description = "The private key in JSON format, base64 encoded"
+  sensitive   = true
+}
+
+output "pubsub_topic_name" {
```

Comment:
The subscription may be more useful for someone troubleshooting an integration

Reply:
we now have both